packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/single-navigation-menu.js

```diff
@@ -28,7 +29,11 @@ export default function SingleNavigationMenu( {
 					onDuplicate={ handleDuplicate }
 				/>
 			}
-			title={ decodeEntities( menuTitle ) }
+			title={ buildNavigationLabel(
+				navigationMenu?.title,
+				navigationMenu?.id,
+				navigationMenu?.status
```

Comment:
This is starting to seem strange. It's like we should just pass the entire Post 😄

Reply:
I did wonder that too